python/tests/BUILD

```diff
@@ -0,0 +1,13 @@
+cc_test(
+    name = "cc_deps_link_test",
+    size = "small",
+    srcs = [
+        "cc_deps_link_test.cc",
+    ],
+    copts = [
+        "-std=c++14",
+    ],
+    deps = [
+        "@tensorflow_cc_deps//:cc_library",
```

Comment:
Is "//" necessary?

Reply:
Yes, the // is required when referring an external repository in a label—per the [Bazel docs on labels](https://bazel.build/concepts/labels) and as verified by trying it out.